Exiled.API/Features/Player.cs

```diff
@@ -534,7 +534,7 @@ public bool IsNoClipEnabled
         /// <summary>
         /// Gets a value indicating whether or not the player is dead.
         /// </summary>
-        public bool IsDead => Role?.Team is Team.Dead;
+        public bool IsDead => Role?.IsDead ?? false;
```

Comment:
Do we still need this?

Reply:
It's a bit redundant but it's nice to have a quick `Player.IsDead` check for certain things (eg. `Player.List(ply => ply.IsDead)`).